homeassistant/components/plugwise/switch.py

```diff
@@ -78,7 +78,7 @@ def __init__(
     @property
     def is_on(self) -> bool | None:
         """Return True if entity is on."""
-        return self.device["switches"].get(self.entity_description.key)
+        return self.device["switches"].get(self.entity_description.key)  # type: ignore [return-value]  # typing through #92253
```

Comment:
Again, you should be able to refactor this in a preliminary PR (as opposed to a follow-up PR)